plugins/kubernetes/test/models/watchers/cluster_pod_watcher_test.rb

```diff
@@ -0,0 +1,220 @@
+require_relative '../../test_helper'
+require 'celluloid/current'
+require 'kubeclient'
+
+class DummyClient
```

Comment:
Normally we'd put these support classes either into their own files under 'test/support', or else put them down below the tests so that we see the tests first.